### PR TITLE
werft/observability: Pin default observability branch to v0.0.1

### PR DESCRIPTION
**Pull request by ArthurSens**
## Description

We're doing a [major refactoring of the observability repo](https://github.com/gitpod-io/observability/pull/45). This PR pins the default branch used to clone the observability repo so we can safely merge breaking changes to main

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **laushinka** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @laushinka` in a comment when ready.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ArthurSens** [comment]
Build is failing but is due ws-manager being stuck on crashloop, making the job timeout. 😢

**Review by mads-hartmann — APPROVED** [review]
LGTM. Added the hold label so we can control the timing of merging this ☺️

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 5f11024a7ae6f7c1ae220419049e76d0677a17b8</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7154#pullrequestreview-828133186" title="Approved">mads-hartmann</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **laushinka** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @laushinka` in a comment when ready.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

Approval requirements bypassed by manually added approval.

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7154#pullrequestreview-828133186" title="Approved">mads-hartmann</a>*

*No associated issue*. Requirement bypassed by manually added approval.

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
